tape/src/main/java/com/squareup/tape2/QueueFile.java

```diff
@@ -113,21 +116,7 @@
 
   @Private boolean closed;
 
-  /**
-   * Constructs a new queue backed by the given file. Only one instance should access a given file
-   * at a time.
-   */
-  public QueueFile(File file) throws IOException {
-    this(file, true);
-  }
-
-  /**
-   * Constructs a new queue backed by the given file. Only one instance should access a given file
-   * at a time.
-   *
-   * @param zero When true, removing an element will also overwrite data with zero bytes.
-   */
-  public QueueFile(File file, boolean zero) throws IOException {
+  QueueFile(File file, boolean zero) throws IOException {
```

Comment:
I think we should go even further and kill this constructor. Move `initializeFromFile` to `build()` and pass in a RAF.